src/server/api/routes/vesselReport.router.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
```

Comment:
Please rename to `vessel-reports.router.js`. Note that I made it plural, since we are doing the same with the url paths.

Reply:
Renamed the file from vesselReport.router.js to vessel-reports.router.js